magento/pwa-studio
# Refactor checkout workflow to set shipping address/get shipping methods with graphql

## Description

Uses graphQL to set shipping address and, in the same mutation, receive the available shipping methods for the selected address.

Also removes email from forms for authed users as the email is already attached to cart.
For guest users it will set the guest email on the cart when submitting the address form.

## Related Issue





Closes [PWA-145](https://jira.corp.magento.com/browse/PWA-145).

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Go through checkout -- it should complete successfully. Verify that the REST submissions still work.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
